modules/histoqc.nf

```diff
@@ -1,11 +1,11 @@
 process HISTOQC {
 
-    container 'ghcr.io/mc2-center/nf-histoqc:latest'
+    container 'ghcr.io/adamjtaylor/nf-histoqc'
```

Comment:
Using old container as mc2-center package is currently private and needs being made public